utils/_context/_scenarios/__init__.py

```diff
@@ -625,10 +625,14 @@ class _Scenarios:
     tracing_config_nondefault_4 = EndToEndScenario(
         "TRACING_CONFIG_NONDEFAULT_4",
         weblog_env={
+            "DD_DYNAMIC_INSTRUMENTATION_ENABLED": "1",
```

Comment:
Just to keep it consistent with the other booleans below:

```suggestion
            "DD_DYNAMIC_INSTRUMENTATION_ENABLED": "true",
```